buildpipeline/pipeline.json

```diff
@@ -16,69 +16,7 @@
       },
       "Definitions": [
         {
-          "Name": "Core-Setup-Linux-BT",
-          "Parameters": {
-            "PB_DistroRid": "debian.8-x64",
-            "PB_DockerTag": "debian-8.2-debpkg-9f87c3c-20173003023006",
-            "REPO_ID": "579f8fb0fedca9aeeb399132",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "Debian 8.2",
-            "Type": "build/product/",
-            "Platform": "x64"
-          }
-        },
-        {
-          "Name": "Core-Setup-Linux-BT",
-          "Parameters": {
-            "PB_DistroRid": "ubuntu.14.04-x64",
-            "PB_DockerTag": "ubuntu-14.04-debpkg-e5cf912-20175003025046",
-            "REPO_ID": "562fbfe0b2d7d0e0a43780c4",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net",
-            "CLI_NUGET_FEED_URL": "https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v2/package",
-            "CLI_NUGET_SYMBOLS_FEED_URL": "https:%2F%2Fdotnet.myget.org/F/dotnet-core/symbols/api/v2/package"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "Ubuntu 14.04",
-            "Type": "build/product/",
-            "Platform": "x64"
-          }
-        },
-        {
-          "Name": "Core-Setup-Linux-BT",
-          "Parameters": {
-            "PB_DistroRid": "ubuntu.16.04-x64",
-            "PB_DockerTag": "ubuntu-16.04-debpkg-e5cf912-20174703024721",
-            "REPO_ID": "575f40f3797ef7280505232f",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "Ubuntu 16.04",
-            "Type": "build/product/",
-            "Platform": "x64"
-          }
-        },
-        {
-          "Name": "Core-Setup-Linux-BT",
-          "Parameters": {
-            "PB_DistroRid": "ubuntu.16.10-x64",
-            "PB_DockerTag": "ubuntu-16.10-debpkg-ec863bb-20170003030028",
-            "REPO_ID": "575f40f3797ef7280505232f",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "Ubuntu 16.10",
-            "Type": "build/product/",
-            "Platform": "x64"
-          }
-        },
-        {
-          "Name": "Core-Setup-Linux-BT",
```

Comment:
I see it is still being used for the ARM build.  Can we rename the files so they are consistent with what we will have for windows after #2311 is merged?


`Core-Setup-Windows-Arm-BT`
`Core-Setup-Windows-BT`

`Core-Setup-Linux-Arm-BT`
`Core-Setup-Linux-BT`

I think that will make more sense going forward.

Unless it is possible to refactor the linux builds into a single build definition for both x64 and arm.

---
In reply to: [116242465](https://github.com/dotnet/core-setup/pull/2374#discussion_r116242465) [](ancestors = 116242465)